demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -903,7 +905,23 @@ export class DemoMeetingApp implements
         this.statsCollector.showDownstreamStats(tileIndex);
       }
     }
+  }
 
+  async getRelayProtocol() {
+    const rawStats = await this.audioVideo.getRTCPeerConnectionStats();
+    if (rawStats) {
+      rawStats.forEach(report => {
+        if (report.type === 'local-candidate') {
+          this.log(`Local WebRTC Ice Candidate stats: ${JSON.stringify(report)}`);
+          let relayProtcol = report.relayProtocol;
```

Comment:
- Typo: `relayProtcol` => `relayProtocol`
- Use `const`
- `relayProtocol` is optional, and calling `toUpperCase()` can throw an error. Check if the type is string.
  https://developer.mozilla.org/en-US/docs/Web/API/RTCIceCandidateStats

```
if (typeof relayProtcol === 'string') {
```

Reply:
Fixed